doc/nrf/libraries/caf/buttons_pm_keep_alive.rst

```diff
@@ -0,0 +1,26 @@
+.. _caf_buttons_pm_keep_alive:
+
+CAF: Power manager keep alive module for buttons
+############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |buttons_pm_keep_alive| of the :ref:`lib_caf` (CAF) is responsible for generating :c:struct:`keep_alive_event` events on any configured button press.
+
+Configuration
+*************
+
+To use the module, you must enable the following Kconfig options:
+
+* :kconfig:`CAF_BUTTONS_PM_KEEP_ALIVE` - This option enables the |buttons_pm_keep_alive|.
+* :kconfig:`CONFIG_CAF_BUTTONS` - This option enables the buttons module.
```

Comment:
I wonder if we should depend on CAF buttons or on CAF button event here.
The module actually reacts also on button_events that can be provided by any module, CAF buttons is not necessary.

Reply:
Currently buttons_pm_keep_alive shares the configuration with buttons. Should I prepare another configuration that depends only buttons events instead of buttons module?

Reply:
Other modules related to PM already share configuration with the main CAF module related to given functionality too. Let's leave it as it is for now.